src/poke_env/player/player.py

```diff
@@ -692,7 +694,9 @@ async def _ladder(self, n_games: int):
             perf_counter() - start_time,
         )
 
-    async def battle_against(self, opponent: "Player", n_battles: int = 1):
+    async def battle_against(
```

Comment:
Can you update the docstring?

Reply:
Does this make `n_battles` a required named argument? If so i'd prefer `opponent` to be either a player or a list of players, and keep `n_battles` from requiring to be named.